plenario/api/point.py

```diff
@@ -154,17 +113,28 @@ def meta():
 
 def _timeseries(args):
 
-    geom = args.data['geom']
-    dataset = args.data.get('dataset')
-    table_names = args.data['dataset_name__in']
-    start_date = args.data['obs_date__ge']
-    end_date = args.data['obs_date__le']
-    agg = args.data['agg']
-
-    # if a single dataset was provided, it's the only thing we need to consider
-    if dataset is not None:
-        table_names = [dataset.name]
-        # for the query's meta information, so that it doesn't show the index
+    meta_params = ['geom', 'dataset', 'dataset_name__in', 'obs_date__ge', 'obs_date__le', 'agg']
+    meta_vals = [args.data.get(k) for k in meta_params]
+    geom, dataset, table_names, start_date, end_date, agg = meta_vals
+
+    ctrees = {}
+
+    if has_tree_filters(args.data):
```

Comment:
builds dataset specific conditions to be applied in the `timseries_all` call if `__filter` params are provided